test/fuzz-regressions/fuzz-unit-file/oss-fuzz-6897

```diff
@@ -0,0 +1,4 @@
+service
+[Service]
+Slice=%H.slice
```

Comment:
Could you change `%H.slice` to `abc-def.slice`? This should make the crash reproducible:
```sh
root@2ca2ae9cdfdc:/src/systemd# cat CRASH
service
[Service]
Slice=abc-def.slice
TemporaryFileSystem=%c

root@2ca2ae9cdfdc:/src/systemd# /out/fuzz-unit-file CRASH
INFO: Seed: 2479253360
INFO: Loaded 2 modules   (51132 inline 8-bit counters): 32527 [0x7f3f8b358a49, 0x7f3f8b360958), 18605 [0xe08248, 0xe0caf5),
INFO: Loaded 2 PC tables (51132 PCs): 32527 [0x7f3f8b360958,0x7f3f8b3dfa48), 18605 [0xe0caf8,0xe555c8),
/out/fuzz-unit-file: Running 1 inputs 1 time(s) each.
Running: CRASH
==21==WARNING: MemorySanitizer: use-of-uninitialized-value
    #0 0x7f3f8ad8b604 in unit_name_is_valid /work/build/../../src/systemd/src/basic/unit-name.c:60:13
    #1 0x7f3f8ac50ad7 in cg_slice_to_path /work/build/../../src/systemd/src/basic/cgroup-util.c:1985:22
    #2 0x645cfc in unit_default_cgroup_path /work/build/../../src/systemd/src/core/cgroup.c:1344:21
    #3 0x5d4a90 in specifier_cgroup /work/build/../../src/systemd/src/core/unit-printf.c:97:21
    #4 0x84db34 in specifier_printf /work/build/../../src/systemd/src/shared/specifier.c:83:45
    #5 0x5d4206 in unit_full_printf /work/build/../../src/systemd/src/core/unit-printf.c:261:16
    #6 0x4edafd in config_parse_temporary_filesystems /work/build/../../src/systemd/src/core/load-fragment.c:4232:21
    #7 0x7f87da in next_assignment /work/build/../../src/systemd/src/shared/conf-parser.c:155:32
    #8 0x7ec9bf in parse_line /work/build/../../src/systemd/src/shared/conf-parser.c:273:16
    #9 0x7ea97c in config_parse /work/build/../../src/systemd/src/shared/conf-parser.c:390:21
    #10 0x4b2ef0 in LLVMFuzzerTestOneInput /work/build/../../src/systemd/src/fuzz/fuzz-unit-file.c:48:16
    #11 0x8a7e05 in fuzzer::Fuzzer::ExecuteCallback(unsigned char const*, unsigned long) /src/libfuzzer/FuzzerLoop.cpp:517:13
    #12 0x861482 in fuzzer::RunOneTest(fuzzer::Fuzzer*, char const*, unsigned long) /src/libfuzzer/FuzzerDriver.cpp:280:6
    #13 0x87388f in fuzzer::FuzzerDriver(int*, char***, int (*)(unsigned char const*, unsigned long)) /src/libfuzzer/FuzzerDriver.cpp:703:9
    #14 0x8605c1 in main /src/libfuzzer/FuzzerMain.cpp:20:10
    #15 0x7f3f8987882f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2082f)
    #16 0x433b38 in _start (/out/fuzz-unit-file+0x433b38)

  Uninitialized value was created by an allocation of 'vla' in the stack frame of function 'cg_slice_to_path'
    #0 0x7f3f8ac504c0 in cg_slice_to_path /work/build/../../src/systemd/src/basic/cgroup-util.c:1942

SUMMARY: MemorySanitizer: use-of-uninitialized-value /work/build/../../src/systemd/src/basic/unit-name.c:60:13 in unit_name_is_valid
Exiting
```

Reply:
I opened https://github.com/google/sanitizers/issues/926.